stackslib/src/core/tests/mod.rs

```diff
@@ -1446,6 +1444,7 @@ fn mempool_db_load_store_replace_tx(#[case] behavior: MempoolCollectionBehavior)
             &mut chainstate,
             &ConsensusHash([0x1; 20]),
             &BlockHeaderHash([0x2; 32]),
+            false,
```

Comment:
Why is `false` necessary in these tests?

Reply:
Because these consensus hashes and block header hashes don't exist in the chainstate (so we tell this non-public function to not try and resolve them to a tenure-start block)